<ui version="4.0" >
 <class>DigitalWindow</class>
 <widget class="QMainWindow" name="DigitalWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QFrame" name="sinkFrame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>800</width>
        <height>500</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <layout class="QHBoxLayout" name="sinkLayout" />
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <spacer name="verticalSpacer" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" >
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="sizeConstraint" >
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="rxBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>250</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>250</width>
          <height>190</height>
         </size>
        </property>
        <property name="title" >
         <string>Receiver Parameters</string>
        </property>
        <widget class="QLineEdit" name="gainMuEdit" >
         <property name="geometry" >
          <rect>
           <x>120</x>
           <y>120</y>
           <width>113</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="gainMuLabel" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>120</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text" >
          <string>mu's gain</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="alphaEdit" >
         <property name="geometry" >
          <rect>
           <x>120</x>
           <y>150</y>
           <width>113</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="alphaLabel" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>150</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text" >
          <string>Alpha</string>
         </property>
        </widget>
        <widget class="QLabel" name="gainLabel" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>60</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text" >
          <string>Gain (dB)</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="freqEdit" >
         <property name="geometry" >
          <rect>
           <x>120</x>
           <y>30</y>
           <width>113</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="freqLabel" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>30</y>
           <width>141</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text" >
          <string>Frequency (Hz)</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="gainEdit" >
         <property name="geometry" >
          <rect>
           <x>120</x>
           <y>60</y>
           <width>113</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="decimLabel" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>90</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text" >
          <string>Decimation</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="decimEdit" >
         <property name="geometry" >
          <rect>
           <x>120</x>
           <y>90</y>
           <width>113</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <widget class="QGroupBox" name="rxPacketBox" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>250</width>
            <height>130</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>250</width>
            <height>130</height>
           </size>
          </property>
          <property name="font" >
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="title" >
           <string>Received Packet Info</string>
          </property>
          <widget class="QLineEdit" name="pktsRcvdEdit" >
           <property name="geometry" >
            <rect>
             <x>120</x>
             <y>30</y>
             <width>113</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="pktsRcvdLabel" >
           <property name="geometry" >
            <rect>
             <x>10</x>
             <y>30</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text" >
            <string>Packets Rcvd.</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="pktsCorrectEdit" >
           <property name="geometry" >
            <rect>
             <x>120</x>
             <y>60</y>
             <width>113</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="pktsCorrectLabel" >
           <property name="geometry" >
            <rect>
             <x>10</x>
             <y>60</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text" >
            <string>Packets Correct</string>
           </property>
          </widget>
          <widget class="QLabel" name="perLabel" >
           <property name="geometry" >
            <rect>
             <x>10</x>
             <y>90</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text" >
            <string>PER</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="perEdit" >
           <property name="geometry" >
            <rect>
             <x>120</x>
             <y>90</y>
             <width>113</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
          <zorder>pktsRcvdEdit</zorder>
          <zorder>pktsRcvdLabel</zorder>
          <zorder>pktsCorrectEdit</zorder>
          <zorder>pktsCorrectLabel</zorder>
          <zorder>perLabel</zorder>
          <zorder>perEdit</zorder>
          <zorder>rxBox</zorder>
          <zorder>verticalLayoutWidget</zorder>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>60</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" >
        <item>
         <spacer name="verticalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="text" >
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>closeButton</zorder>
   <zorder>sinkFrame</zorder>
   <zorder>rxBox</zorder>
   <zorder>rxPacketBox</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>DigitalWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>617</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>DigitalWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>960</x>
     <y>694</y>
    </hint>
    <hint type="destinationlabel" >
     <x>66</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
